<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PokemonBag</class>
 <widget class="QWidget" name="PokemonBag">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>960</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>110</y>
     <width>581</width>
     <height>751</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>579</width>
      <height>749</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>450</y>
     <width>191</width>
     <height>421</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="7" column="1">
     <widget class="QLabel" name="hp">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>类型：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="kind">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="label_9">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>技能：</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="label_10">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生命值：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="label_7">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>敏捷：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="def">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>经验值：</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_8">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>防御：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>属性：</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="agl">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>等级：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="type">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>名字：</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="exp">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="name">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="atk">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>攻击：</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLabel" name="skill">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="level">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="label_11">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>攻击间隔：</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLabel" name="delay">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>84</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>拥有徽章：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>757</x>
     <y>330</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>胜率：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>100</y>
     <width>291</width>
     <height>161</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="badgeNumber">
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>77</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="badgeHigh">
      <property name="maximumSize">
       <size>
        <width>55</width>
        <height>77</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="badgeNumberInfo">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="badgeHighInfo">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>320</y>
     <width>291</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="battleInfo">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="winInfo">
      <property name="font">
       <font>
        <family>微软雅黑</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
